tests/lib/rules/jsx-uses-vars.js

```diff
@@ -115,6 +115,22 @@ ruleTester.run('no-unused-vars', rule, {
     }, {
       code: '\
         /*eslint jsx-uses-vars:1*/\
+        var App;\
+        var Hello;\
+        React.render(<App:Hello/>);',
```

Comment:
I'm not at all familiar with this namespace syntax, so I'm not really sure if this is correctly marked as an error or not. @ljharb can you verify?